index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Dmitrij Krutalianau</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <base target="_blank">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="page">
+    <main>
+        <h1>Summary of Dima Krutolianov</h1>
+        <div class="container" id="introduction">
+            <div class="header">
+                <h2>Introduction</h2>
```

Comment:
You could apply styles for h2 element, <div class="header"> is not required. Here and below.

Reply:
@OlgaPoddubko, Yeah, see it now. I introduced it at the beginning and didn't check later